### ISSUE TITLE
Profile view: List of requests doesn't update after changing approver employee

### ISSUE THREAD

**Issue by KuchaLeni** [SoftwareCountry/arcadian-assistant#401]
1. Log in as employee of the department
2. Go to the calendar view 
3. Request vacation/dayoff/sick leave
4. Log out and log in as chief of department from step 1
5. Note that request from the step 3 is displayed in the Profile view
6. Go to DB and change chief of the department from the step 1
7. Log in as user from the step 4
8. Go to the Profile view

**Result:**  Request from the step 3 is displayed in the Profile view (Request is not displayed in requests list of the new department chief)
**Expected result:**  Request from the step 3 is not displayed in the Profile view. Requests from the step 3 is displayed in request list of the new department chief.

**Comment by sponkrashin** [comment]
Yes, for now we doesn't handle such situations. Approver is selected once after event was created or previous approval was granted. Approver will be changed only after server part restarts. This is expected behavior. Am I right, @santee?

**Comment by sponkrashin** [comment]
We discussed with @santee and decided to postpone it for some time.
